src/scripts/modules/media/body/body.coffee

```diff
@@ -125,12 +134,201 @@ define (require) ->
           $temp.find('ol[start], [data-type="list"][data-list-type="enumerated"][start]').each (i, el) ->
             $el = $(el)
             $el.css('counter-reset', 'list-item ' + $el.attr('start'))
+
+          # # uncomment to embed fake exercises and see embeddable exercises in action
+          # @fakeExercises($temp)
+
+          @initializeEmbeddableQueues()
+          @findEmbeddables($temp.find('#content'))
+
       catch error
         # FIX: Log the error
         console.log error
 
       @$el?.html($temp.html())
 
+
+
+    ###
+    # For handling embeddables (exercises from Tutor, concept coaches, simulations, etc).
+    # Could abstract a little more and pull out to another file if needed.
+    ###
+
+    # A queue for rendering embeddables means
+    # we can share rendering functions regardless
+    # of whether the embeddable is sync or async
+    initializeEmbeddableQueues: () ->
+      @renderEmbeddableQueue = []
+
+      _.extend(@renderEmbeddableQueue, Backbone.Events)
```

Comment:
I haven't looked at this closely to comment on it intelligently, but as a quick glance this strikes me as something that needs a closer look as possibly being overkill.